packages/database/src/models/AccessLog.ts

```
@@ -0,0 +1,81 @@
+
+      {
+        ...options,
+        tableName: 'accesses',
+        syncDirection: SYNC_DIRECTIONS.DO_NOT_SYNC,
+        schema: 'logs',
+      },
+    );
+  }
+
+  static initRelations(models: Models) {
+    this.belongsTo(models.User, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)